### PR TITLE
Ingestion csvs for moorings recovered on P16

### PR DESCRIPTION
**Pull request by cdobs**
Created recovered ingestion csvs for moorings recovered on Pioneer 16:
-CP01CNPM R00004
-CP01CNSM R00013
-CP02PMCI R00014
-CP02PMCO R00014
-CP02PMUI R00015
-CP02PMUO R00016
-CP03ISPM R00004
-CP03ISSM R00012
-CP04OSPM R00013
-CP04OSSM R00012

### CODE REVIEW

File: CP04OSPM/CP04OSPM_R00013_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP04OSPM/R00013/instruments/CWFP_sn*/*/E*.DA?,CP04OSPM-WFP01-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_k.wfp.vel3d_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP04OSPM/R00013/instruments/CWFP_sn*/*/A*.DA?,CP04OSPM-WFP01-01-VEL3DK000,recovered_wfp,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP04OSPM/R00013/instruments/CWFP_sn*/*/C*.DA?,CP04OSPM-WFP01-02-DOFSTK000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP04OSPM/R00013/instruments/CWFP_sn*/*/C*.DA?,CP04OSPM-WFP01-03-CTDPFK000,recovered_wfp,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP04OSPM/R00013/instruments/CWFP_sn*/*/E*.DA?,CP04OSPM-WFP01-04-FLORTK000,recovered_wfp,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP04OSPM/R00013/instruments/CWFP_sn*/*/E*.DA?,CP04OSPM-WFP01-05-PARADK000,recovered_wfp,Available,
```
Comment (snwhite) [inline CP04OSPM/CP04OSPM_R00013_ingest.csv:11]: Looks good.

File: CP03ISPM/CP03ISPM_R00004_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP03ISPM/R00004/instruments/CWFP_sn*/*/E*.DA?,CP03ISPM-WFP01-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_k.wfp.vel3d_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP03ISPM/R00004/instruments/CWFP_sn*/*/A*.DA?,CP03ISPM-WFP01-01-VEL3DK000,recovered_wfp,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP03ISPM/R00004/instruments/CWFP_sn*/*/C*.DA?,CP03ISPM-WFP01-02-DOFSTK000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP03ISPM/R00004/instruments/CWFP_sn*/*/C*.DA?,CP03ISPM-WFP01-03-CTDPFK000,recovered_wfp,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP03ISPM/R00004/instruments/CWFP_sn*/*/E*.DA?,CP03ISPM-WFP01-04-FLORTK000,recovered_wfp,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP03ISPM/R00004/instruments/CWFP_sn*/*/E*.DA?,CP03ISPM-WFP01-05-PARADK000,recovered_wfp,Available,
```
Comment (snwhite) [inline CP03ISPM/CP03ISPM_R00004_ingest.csv:11]: Looks good.

File: CP02PMUO/CP02PMUO_R00016_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMUO/R00016/instruments/CWFP_sn*/*/E*.DA?,CP02PMUO-WFP01-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_k.wfp.vel3d_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMUO/R00016/instruments/CWFP_sn*/*/A*.DA?,CP02PMUO-WFP01-01-VEL3DK000,recovered_wfp,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMUO/R00016/instruments/CWFP_sn*/*/C*.DA?,CP02PMUO-WFP01-02-DOFSTK000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMUO/R00016/instruments/CWFP_sn*/*/C*.DA?,CP02PMUO-WFP01-03-CTDPFK000,recovered_wfp,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMUO/R00016/instruments/CWFP_sn*/*/E*.DA?,CP02PMUO-WFP01-04-FLORTK000,recovered_wfp,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMUO/R00016/instruments/CWFP_sn*/*/E*.DA?,CP02PMUO-WFP01-05-PARADK000,recovered_wfp,Available,
```
Comment (snwhite) [inline CP02PMUO/CP02PMUO_R00016_ingest.csv:11]: Looks good.

File: CP02PMUI/CP02PMUI_R00015_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+#mi.dataset.driver.vel3d_k.wfp.vel3d_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMUI/R00015/instruments/CWFP_sn*/*/A*.DA?,CP02PMUI-WFP01-01-VEL3DK000,recovered_wfp,Not Expected,WFP not recovered
+#mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMUI/R00015/instruments/CWFP_sn*/*/C*.DA?,CP02PMUI-WFP01-02-DOFSTK000,recovered_wfp,Not Expected,WFP not recovered
+#mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMUI/R00015/instruments/CWFP_sn*/*/C*.DA?,CP02PMUI-WFP01-03-CTDPFK000,recovered_wfp,Not Expected,WFP not recovered
+#mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMUI/R00015/instruments/CWFP_sn*/*/E*.DA?,CP02PMUI-WFP01-04-FLORTK000,recovered_wfp,Not Expected,WFP not recovered
+#mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMUI/R00015/instruments/CWFP_sn*/*/E*.DA?,CP02PMUI-WFP01-05-PARADK000,recovered_wfp,Not Expected,WFP not recovered
```
Comment (snwhite) [inline CP02PMUI/CP02PMUI_R00015_ingest.csv:11]: Looks good.

**Review by snwhite — APPROVED** [review]
All files look ok.

File: CP01CNSM/CP01CNSM_R00013_ingest.csv [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00013/cg_data/dcl12/hyd2/*.hyd2.log,CP01CNSM-SBD12-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00013/cg_data/dcl12/pco2a/*pco2a.log,CP01CNSM-SBD12-04-PCO2AA000,recovered_host,Available,
+mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00013/cg_data/dcl12/wavss/*.wavss.log,CP01CNSM-SBD12-05-WAVSSA000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00013/cg_data/dcl12/metbk2/*metbk2.log,CP01CNSM-SBD12-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.fdchp_a.dcl.fdchp_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00013/cg_data/dcl12/fdchp/*fdchp.log,CP01CNSM-SBD12-08-FDCHPA000,recovered_host,Available,
+mi.dataset.driver.fdchp_a.fdchp_a_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00013/instruments/dcl12/FDCHP_sn*/D20*/fdchp*.dat,CP01CNSM-SBD12-08-FDCHPA000,recovered_inst,Available,
```
Comment (asmith-whoi) [inline CP01CNSM/CP01CNSM_R00013_ingest.csv:46]: OK

File: CP01CNPM/CP01CNPM_R00004_ingest.csv [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00004/instruments/CWFP_sn*/*/E*.DA?,CP01CNPM-WFP01-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_k.wfp.vel3d_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00004/instruments/CWFP_sn*/*/A*.DA?,CP01CNPM-WFP01-01-VEL3DK000,recovered_wfp,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00004/instruments/CWFP_sn*/*/C*.DA?,CP01CNPM-WFP01-02-DOFSTK000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00004/instruments/CWFP_sn*/*/C*.DA?,CP01CNPM-WFP01-03-CTDPFK000,recovered_wfp,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00004/instruments/CWFP_sn*/*/E*.DA?,CP01CNPM-WFP01-04-FLORTK000,recovered_wfp,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP01CNPM/R00004/instruments/CWFP_sn*/*/E*.DA?,CP01CNPM-WFP01-05-PARADK000,recovered_wfp,Available,
```
Comment (asmith-whoi) [inline CP01CNPM/CP01CNPM_R00004_ingest.csv:11]: OK

File: CP02PMCI/CP02PMCI_R00014_ingest.csv [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00014/instruments/CWFP_sn*/*/E*.DA?,CP02PMCI-WFP01-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_k.wfp.vel3d_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00014/instruments/CWFP_sn*/*/A*.DA?,CP02PMCI-WFP01-01-VEL3DK000,recovered_wfp,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00014/instruments/CWFP_sn*/*/C*.DA?,CP02PMCI-WFP01-02-DOFSTK000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00014/instruments/CWFP_sn*/*/C*.DA?,CP02PMCI-WFP01-03-CTDPFK000,recovered_wfp,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00014/instruments/CWFP_sn*/*/E*.DA?,CP02PMCI-WFP01-04-FLORTK000,recovered_wfp,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCI/R00014/instruments/CWFP_sn*/*/E*.DA?,CP02PMCI-WFP01-05-PARADK000,recovered_wfp,Available,
```
Comment (asmith-whoi) [inline CP02PMCI/CP02PMCI_R00014_ingest.csv:11]: OK

File: CP02PMCO/CP02PMCO_R00014_ingest.csv [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCO/R00014/instruments/CWFP_sn*/*/E*.DA?,CP02PMCO-WFP01-00-WFPENG000,recovered_wfp,Available,
+mi.dataset.driver.vel3d_k.wfp.vel3d_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCO/R00014/instruments/CWFP_sn*/*/A*.DA?,CP02PMCO-WFP01-01-VEL3DK000,recovered_wfp,Available,
+mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCO/R00014/instruments/CWFP_sn*/*/C*.DA?,CP02PMCO-WFP01-02-DOFSTK000,recovered_wfp,Available,
+mi.dataset.driver.ctdpf_ckl.wfp.coastal_ctdpf_ckl_wfp_recovered_driver,/omc_data/whoi/OMC/CP02PMCO/R00014/instruments/CWFP_sn*/*/C*.DA?,CP02PMCO-WFP01-03-CTDPFK000,recovered_wfp,Available,
+mi.dataset.driver.flort_kn.stc_imodem.flort_kn__stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCO/R00014/instruments/CWFP_sn*/*/E*.DA?,CP02PMCO-WFP01-04-FLORTK000,recovered_wfp,Available,
+mi.dataset.driver.PARAD_K.STC_IMODEM.parad_k_stc_imodem_recovered_driver,/omc_data/whoi/OMC/CP02PMCO/R00014/instruments/CWFP_sn*/*/E*.DA?,CP02PMCO-WFP01-05-PARADK000,recovered_wfp,Available,
```
Comment (asmith-whoi) [inline CP02PMCO/CP02PMCO_R00014_ingest.csv:11]: OK

File: CP02PMUO/CP02PMUO_R00016_ingest.csv (hunk 3 above)
Comment (asmith-whoi) [inline CP02PMUO/CP02PMUO_R00016_ingest.csv:11]: OK

File: CP03ISPM/CP03ISPM_R00004_ingest.csv (hunk 2 above)
Comment (asmith-whoi) [inline CP03ISPM/CP03ISPM_R00004_ingest.csv:11]: OK

File: CP03ISSM/CP03ISSM_R00012_ingest.csv [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00012/cg_data/dcl11/mopak/*.mopak.log,CP03ISSM-SBD11-01-MOPAK0000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00012/cg_data/dcl11/hyd1/*.hyd1.log,CP03ISSM-SBD11-02-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00012/cg_data/dcl11/metbk/*.metbk.log,CP03ISSM-SBD11-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00012/cg_data/dcl12/syslog/*.syslog.log,CP03ISSM-SBD12-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00012/cg_data/dcl12/hyd2/*.hyd2.log,CP03ISSM-SBD12-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00012/cg_data/dcl12/pco2a/*pco2a.log,CP03ISSM-SBD12-04-PCO2AA000,recovered_host,Available,
```
Comment (asmith-whoi) [inline CP03ISSM/CP03ISSM_R00012_ingest.csv:42]: OK

File: CP04OSPM/CP04OSPM_R00013_ingest.csv (hunk 1 above)
Comment (asmith-whoi) [inline CP04OSPM/CP04OSPM_R00013_ingest.csv:11]: OK

File: CP02PMUI/CP02PMUI_R00015_ingest.csv (hunk 4 above)
Comment (asmith-whoi) [inline CP02PMUI/CP02PMUI_R00015_ingest.csv:11]: OK

File: CP04OSSM/CP04OSSM_R00012_ingest.csv [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00012/cg_data/dcl11/mopak/*.mopak.log,CP04OSSM-SBD11-01-MOPAK0000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00012/cg_data/dcl11/hyd1/*.hyd1.log,CP04OSSM-SBD11-02-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00012/cg_data/dcl11/metbk/*.metbk.log,CP04OSSM-SBD11-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00012/cg_data/dcl12/syslog/*.syslog.log,CP04OSSM-SBD12-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00012/cg_data/dcl12/hyd2/*.hyd2.log,CP04OSSM-SBD12-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00012/cg_data/dcl12/pco2a/*pco2a.log,CP04OSSM-SBD12-04-PCO2AA000,recovered_host,Available,
```
Comment (asmith-whoi) [inline CP04OSSM/CP04OSSM_R00012_ingest.csv:42]: OK

**Review by asmith-whoi — APPROVED** [review]

**Pull request merged**